src/modules/transaction/subscriptions/newTransactionSubscription.ts

```diff
@@ -0,0 +1,32 @@
+import { subscriptionWithClientId } from "graphql-relay-subscription"
+import { pubSub } from "../../../PubSub"
+
+import TransactionModel from "../TransactionModel"
+import TransactionType from "../TransactionType"
+
+type newTransaction = {
+  transactionId: string
+}
+
+const TransactionNewSubscription = subscriptionWithClientId({
+  name: "TransactionNew",
+  inputFields: {},
+  outputFields: {
+    transaction: {
+      type: TransactionType,
+      resolve: async ({id}: any) => TransactionModel.findOne({_id: id})
```

Comment:
Why are your type `any`?

Reply:
Actually I just forgot this one. Fixed on this commit here: https://github.com/LeAlencar/relay-graphql-api/commit/473557cd4a99934e85a2de19cb07d5715ce4c387

Reply:
Nice one